app/src/main/res/layout/search_dialog.xml

```diff
@@ -33,6 +32,31 @@
         android:maxLines="1"
         android:singleLine="true"/>
 
+    <View style="@style/ThinDivider"/>
```

Comment:
I can see those dividers in the IDE but not in the emulator. Have you got an idea why?

Reply:
However, in general the dialog does not look very nice with these dividers. Please add at least some left and right space as in the `RecyclerView`.